src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(m -> Arrays.stream(m.split(",")))
+                .mapToInt(s -> Integer.parseInt(s))
+                .filter(i -> i % 2 == 0)
```

Comment:
This is more understandable, fix in all palces
```suggestion
                .flatMap(stringNumbers -> Arrays.stream(stringNumbers.split(DELIMITER)))
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
```